src/test/test.ts

```diff
@@ -0,0 +1,71 @@
+import { PrismaClient } from "@prisma/client";
+import axios from "axios";
+import { compare } from "bcrypt-ts";
+import { expect } from "chai";
+import "mocha";
+import { start, stop } from "../setup";
+
+let prisma: PrismaClient;
+
+before(async () => {
+  prisma = await start();
+});
+
+after(async () => {
+  await prisma.user.deleteMany();
+  await stop();
```

Comment:
Aqui você criou outra variável pra guardar a instância do prisma, que você usa pra fazer setups e conferir os dados no banco no meio dos testes. Será que não vale então exportar a instância do prisma lá do `setup-db` pra isso? Porque assim quando você for trabalhar com múltiplos arquivos de teste, você não vai precisar ficar criando essa variável repetidamente.